### ISSUE TITLE
Crash with v2.350 and certain AUR builds

### ISSUE THREAD

**Issue by muesli** [Jguer/yay#158]
Trying to install `qownnotes` from AUR:

```
==> Validating source files with sha256sums...
    qownnotes-18.02.3.tar.xz ... Passed
==> Extracting sources...
  -> Extracting qownnotes-18.02.3.tar.xz with bsdtar
==> Sources are ready.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x6435dc]

goroutine 1 [running]:
github.com/mikkeloscar/gopkgbuild.(*PKGBUILD).CompleteVersion(...)
        /home/muesli/yaytmp/yay/src/.go/src/github.com/mikkeloscar/gopkgbuild/pkgbuild.go:145
main.buildInstallPkgBuilds(0xc4200940a0, 0x1, 0x1, 0xc420201568, 0xc4204260f0, 0xc420426120, 0xc420426a80, 0xc42044a210, 0x21)
        /home/muesli/yaytmp/yay/src/yay-2.350/install.go:358 +0xc5c
main.install(0xc420426120, 0xc42033a0c0, 0x1)
        /home/muesli/yaytmp/yay/src/yay-2.350/install.go:149 +0xcec
main.upgradePkgs(0xc420051c40, 0x0, 0x0, 0x2, 0xc420081301)
        /home/muesli/yaytmp/yay/src/yay-2.350/upgrade.go:416 +0x87c
main.handleSync(0xc420092430, 0x1)
        /home/muesli/yaytmp/yay/src/yay-2.350/cmd.go:494 +0x2d0
main.handleCmd(0x0, 0x0)
        /home/muesli/yaytmp/yay/src/yay-2.350/cmd.go:324 +0xb1d
main.main()
        /home/muesli/yaytmp/yay/src/yay-2.350/cmd.go:225 +0x324
```

**Comment by Morganamilo** [comment]
If you don't want to wait for a new release you can it yet when it asks if you want to edit the pkgbuild and change the arch.
```diff
-arch=('i686' 'x86_64' 'armv7l')
+arch=('i686' 'x86_64' 'armv7h')
```

**Comment by Morganamilo** [comment]
The real fix should come along after mikkeloscar/gopkgbuild/12

**Comment by Morganamilo** [comment]
Fixed upstream in mikkeloscar/gopkgbuild#13.
